docusaurus/docs/get-started/get-started.mdx

```diff
@@ -38,15 +38,15 @@ Scaffold a new plugin with a single command! Run the following and answer the pr
 
 ## Why create a Grafana plugin?
 
-Grafana plugin development allows you to create many different types of user experiences using [supported](/introduction/grafana-plugin-sdk-for-go) features. For example, you can make:
+Grafana plugin development allows you to create many different types of user experiences using [supported](/key-concepts/backend-plugins/grafana-plugin-sdk-for-go) features. For example, you can make:
```

Comment:
Not for this PR, but this is a very strange link out to the SDK...

Reply:
indded, let's rewrite this soon (not today)